pkg/arvo/sys/hoon.hoon

```diff
@@ -821,14 +832,14 @@
   ::
 ++  bex                                                 ::  binary exponent
   ~/  %bex
-  |=  a=@
+  |=  a=bloq
   ^-  @
   ?:  =(0 a)  1
   (mul 2 $(a (dec a)))
 ::
 ++  can                                                 ::  assemble
   ~/  %can
-  |=  [a=bloq b=(list [p=@u q=@])]
+  |=  [a=bloq b=(list [p=step q=@])]
```

Comment:
I'd err on the side of keeping these `@u`, it's a count; but not a strong opinion.